core/src/por/api.ts

```diff
@@ -45,6 +54,37 @@ export async function insertData({
       data: response?.data
     }
   } catch (e) {
+    logger.error(`Failed to insert Data. API-URL:${ORAKL_NETWORK_API_URL}, Data: ${data}`)
     throw new OraklError(OraklErrorCode.FailedInsertData)
   }
 }
+
+export async function insertAggregateData({
```

Comment:
I have a question.
Where is this function referenced?

Reply:
@nick-bisonai Inside `fetchWithAggregator`

Reply:
@nick-bisonai Here is the reference function.
https://github.com/Bisonai/orakl/blob/master/fetcher/src/job/job.api.ts#L69

The function is called to store the aggregated data through `orakl-api`.